sublime_jedi/helper.py

```diff
@@ -2,7 +2,22 @@
 import sublime
 import sublime_plugin
 
+import html
+
+try:
+    # mdpopups needs 3119+ for wrapper_class, which diff popup relies on
+    if int(sublime.version()) < 3119:
+        raise ImportError('Sublime Text 3119+ required.')
+    # mdpopups 1.9.0+ is required because of wrapper_class and templates
+    import mdpopups
+    if mdpopups.version() < (1, 9, 0):
+        raise ImportError('mdpopups 1.9.0+ required.')
+    _HAVE_MDPOPUPS = True
+except ImportError:
```

Comment:
I copied this structure from GitGutter's diff popup module which used this kind of "option-handling" as long as I know it. If mdpopups is not available an ImportError is thrown anyway. Therefore I read this construct as a very obvious hint for someone who has a look into the code.

It could be logged, but as long as mdpopups is optional it is not really an error.

I am open for better and cleaner suggestions how to handle optional import of a module including version checks.